### ISSUE TITLE
Animation on a page is resulting into false visual metrics

### ISSUE THREAD

**Issue by amitdisha** [sitespeedio/sitespeed.io#4180]
### Have you read the documentation?

- [X] Yes, I've read the [how to make a reproducable bug guide](https://www.sitespeed.io/documentation/sitespeed.io/bug-report/)
- [X] Yes, I've read the [how to debug my script guide](https://www.sitespeed.io/documentation/sitespeed.io/scripting/#debug)

### URL

NA

### What are you trying to accomplish

Animations on a webpage are indeed interfering with visual performance metrics captured by tools like sitespeed.io.
In this case what does Sitespeed.io recommend ? Need to check if sitespeed.io offers options to ignore or handle animations ?

### What browser did you use?

Chrome

### How to reproduce

```shell
Run sitespeed.io test for any webpage having animation like below css
.jss333 {
    color: #91B9FA;
    width: fit-content;
    border: 1px solid #91B9FA;
    display: flex;
    padding: 0 3px;
    animation: jss334 4s infinite;
    animation-duration: 4s;
    animation-timing-function: ease;
    animation-delay: 0s;
    animation-iteration-count: infinite;
    animation-direction: normal;
    animation-fill-mode: none;
    animation-play-state: running;
    animation-name: jss334;
    animation-timeline: auto;
    animation-range-start: normal;
```

**Comment by soulgalore** [comment]
Hi @amitdisha which of the visual metrics is false? Maybe you could use `--browsertime.injectJs ` to inject some JavaScript that remove/blocks the element?

**Comment by amitdisha** [comment]
Thanks for your quick response.
 I will try above option but i doubt I will be allowed to inject js in production. 
May be will test in lower env Or disable animation as last option in lower env. 
Thanks

**Comment by soulgalore** [comment]
Yeah but what metrics do you mean is false/wrong?

Best
Peter

**Comment by amitdisha** [comment]
Last visual change, visualchange99, 95 , 85.
Actual value when animation is disabled is around 2s. But after enabling
animation they come around 8 to 9s.

**Comment by soulgalore** [comment]
But aren't that correct?

**Comment by amitdisha** [comment]
In my view, animations should be ignored while capturing visual metrics else it gives false alarm that page visual loading is very poor for end user experience.

**Comment by soulgalore** [comment]
Yeah but it's hard to distinguish why content change on the screen when you analyse a video? It would be a metric like lastVisualChangeWithoutCSSAnimations :) I think the workaround need to be to block out that part of the screen. Maybe we could come up with a way to do that more easily.
